wiki/markdown/configuration.md

```diff
@@ -141,6 +141,36 @@ used. **er.url** takes a full URL to such a repository.
 
 * er.url
 
+## RabbitMQ
+
+You can configure the RabbitMQ settings using the rabbitmq.* properties.
+Most of the properties should be familiar but a few may need some further explanation.
+The rabbitmq.domainId, rabbitmq.componentName, rabbitmq.consumerName and rabbitmq.queue.durable are used to build
+the queue name on which Eiffel Intelligence listens for messages.
+The waitlist queue also attaches the rabbitmq.waitlist.queue.suffix property and all of the previously
```

Comment:
Perhaps clarify that this separate (user-configurable) bindnig is for **consuming** events. And that the one mentioned below (routing key which is not user configurable) is only for **sending** events ...

Reply:
I have to test something first. I may have made a misstake in the queue suffix story to investigate its use. I gave it a thumbs up and added it to the docs but just now I realised what my original issue with it was. I don't think the queue is ever actually created in rabbitmq.

Reply:
Ok, so I changed the code so that the waitlist queue is actually created, gets a consumer and a binding. I don't know why this wasn't properly done before but I need to change https://github.com/eiffel-community/eiffel-intelligence/pull/452 as well now.

Reply:
I updated the text a bit. Don't know if it's good.